apps/chat/src/utils/server/api.ts

```diff
@@ -107,10 +107,14 @@ export class ApiUtils {
     }).pipe(
       switchMap((response) => {
         if (!response.ok) {
-          return throwError(() => new Error(response.status + ''));
+          return from(response.text()).pipe(
```

Comment:
I'm not sure that we always will receive some text.
At least it would be nice to add `catch` also